services/newsfeed/config_drupal.js

```diff
@@ -15,51 +15,30 @@
  */
 const getDrupalConfig = ({ enabled, name, url, filter }) => ({
 	name,
-	type: 'rest',
 	enabled,
 	url,
-	interval: 5 * 60, // seconds
+	table: 'newsfeed',
```

Comment:
Kindly consider maintaining the `interval` within the config and referring it within the job.